viz/charts/linkerd-viz/values.yaml

```diff
@@ -61,16 +63,20 @@ metricsAPI:
   # -- number of replicas of the metrics-api component
   replicas: 1
   # -- log level of the metrics-api component
-  logLevel: *log_level
+  # @default -- defaults to defaultLogLevel
```

Comment:
Should we remove this `defaults to` part? 🤔 @alpeb

Reply:
@Pothulapati I'm +1 for removing the `defaults to` for each of these descriptions. The column it's filling is showing the default value, so I think its a little redundant to then also say `defaults to ...`.

Reply:
:+1: